src/CRM/CivixBundle/Resources/views/Code/test-api.php.php

```diff
@@ -21,7 +21,7 @@ class <?php echo $testClassName ?> extends \PHPUnit\Framework\TestCase implement
    *
    * See: https://docs.civicrm.org/dev/en/latest/testing/phpunit/#civitest
    */
-  public function setUpHeadless() {
+  public function setUpHeadless(): void {
```

Comment:
Doesn't make sense on a few fronts:

* The function here actually does return a value, which raises a PHP error.
* This disagrees with `HeadlessInterface`'s signature for `setUpHeadless()`
* Because the templates have it return a value... therefore one should expect extant tests to also return a value.